plugins/inventory/lxd.py

```diff
@@ -0,0 +1,1060 @@
+            - mostly: unix:/var/lib/lxd/unix.socket or unix:/var/snap/lxd/common/lxd/unix.socket
+            required: false
+            type: str
+        client_key:
+            description:
+            - The client certificate key file path.
+            required: false
+            aliases: [ key_file ]
+            default: $HOME/.config/lxc/client.key
+            type: path
+        client_cert:
+            description:
+            - The client certificate file path.
+            required: false
+            aliases: [ cert_file ]
+            default: $HOME/.config/lxc/client.crt
+            type: path
+        trust_password:
+            description:
+            - The client trusted password.
+            - You need to set this password on the LXD server before
+                running this module using the following command.
+                lxc config set core.trust_password <some random password>
+                See (https://www.stgraber.org/2016/04/18/lxd-api-direct-interaction/)
+            - If trust_password is set, this module send a request for
+                authentication before sending any requests.
+            required: false
+            type: str
+        state:
+            description: Filter the container according to the current status
+            required: false
+            type: str
+            default: none
+            choices: ['STOPPED', 'STARTING', 'RUNNING', 'none']
+        prefered_container_network_interface:
+            description: if a container has multiple network interfaces, which one is the prefered as pattern
+            required: false
+            type: str
+            default: eth
+            notes:
+              - combined with the first number that can be found e.g. eth + 0
+        prefered_container_network_family:
+            description: if a container has multiple network interfaces, which one is the prefered by family
+            required: false
+            type: str
+            default: inet
+            choices: ['inet', 'inet6']
+            notes:
+              - inet == ipv4
+              - inet6 == ipv6
+        groups:
+            description: Group the results
+            required: false
+            type: json
+            default: none
+            notes:
+              - lxd location: no idea need sample
+              - pattern: samba* or regex
+              - network_range: 192.168.0.0/24
+              - os: ubuntu
+              - release: groovy
+              - profile: default
+              - vlanid: works on defined networks e.g. br0 with vlan_default == 666
+        selftest:
+            description: Load default data to test plugIn
+            required: false
+            type: bool
+            default: false
+        dumpdata:
+            description: dump out data to debug
+            required: false
+            type: bool
+            default: false
+'''
+
+EXAMPLES = '''
+# simple lxd.yml
+plugin: lxd
```

Comment:
```suggestion
plugin: community.general.lxd
```
This will not work otherwise.

Reply:
done